managed/src/main/resources/ossFeatureConfig.json

```
@@ -40,6 +40,7 @@
         "restartUniverse": "disabled",
         "metricsInterval": 15000,
         "editUniverse": "hidden",
+        "pausedUniverse": "enabled",
```

```suggestion
        "pausedUniverse": "disabled",
```
Sorry, one last thing. Can you disable this option for `ossFeatureConfig.json`? I had to confirm with our team just to make sure.

---

@yb-andrew : Out of curiousity, for read only user we need to disable the option of Pause Universe, once I changed the read only config, it worked for me, can I know why we need to make changes here too, please clarify

---

@rajmaddy89 Sure, I'll merge this PR. This file is not related to your change or the original ticket, but we also want to remove this option for our open source executable, `yugabyted`, because it is out of scope for users testing out the product.